### PR TITLE
Wip/hughsie/ca

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: contrib/ACME-CA.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+organization = "ACME Corp."
+cn = "ACME CA"
+uri = "http://www.example.com/"
+email = "admin@example.com"
+crl_dist_points = "http://www.example.com/pki/"
+serial = 1
+crl_number = 1
+path_len = 1
+activation_date = "2017-08-01 00:00:00Z"
+expiration_date = "2047-08-01 00:00:00Z"
```
Comment (superm1) [inline contrib/ACME-CA.cfg]: To discourage use in production what do you think about setting the default value for activation as the date you run the script and expiration as 30 days later?

File: README.md [hunk 2]
```diff
@@ -63,64 +63,29 @@ instance the LVFS) it does mean it can verify your signatures.
...
+    $ jcat-tool --appstream-id rhughes sign firmware.jcat firmware.bin rhughes_signed.pem rhughes.key
     JcatFile:
       Version:               0.1
       JcatItem:
         ID:                  firmware.bin
         JcatBlob:
           Kind:              pkcs7
           Flags:             is-utf8
-          AppstreamId:       com.dell
+          AppstreamId:       rhughes
```
Comment (superm1) [inline README.md]: Doesn't seem like a valid appsteam id now?

File: contrib/ACME-CA.cfg (hunk 1 above)
Comment (hughsie) [reply contrib/ACME-CA.cfg]: Have something like ACME-CA.cfg.in and then process it with meson? My worry with that would be that when you're testing libjcat might not be the same time that you're building jcat. Or you mean something like injecting the dates into the cfg using the shell script when it builds the cert? Maybe the shell script actually wants to be python if we're doing anything remotely clever...

File: README.md (hunk 2 above)
Comment (hughsie) [reply README.md]: Maybe `com.redhat.rhughes`?

Comment (superm1) [reply README.md]: Yeah

File: contrib/ACME-CA.cfg (hunk 1 above)
Comment (superm1) [reply contrib/ACME-CA.cfg]: I was thinking injecting dates when building certs.

Comment (hughsie) [reply contrib/ACME-CA.cfg]: Something like this?

**Pull request merged**